pavelib/utils/test/suites/bokchoy_suite.py

```diff
@@ -100,12 +100,16 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_value, traceback):
         super(BokChoyTestSuite, self).__exit__(exc_type, exc_value, traceback)
 
-        msg = colorize('green', "Cleaning up databases...")
-        print msg
-
         # Clean up data we created in the databases
-        sh("./manage.py lms --settings bok_choy flush --traceback --noinput")
-        bokchoy_utils.clear_mongo()
+        if not self.testsonly:
```

Comment:
nit: this is a style thing, feel free to disregard. I'd flip the two paths and have "if/else" rather than "if not/else"